src/utils/admin-utils.ts

```diff
@@ -0,0 +1,32 @@
+import { FeatureCollection } from 'geojson';
+
+export function getAdminLayerURL(adminLevel = 1): string {
+  switch (adminLevel) {
+    case 3:
+      return './data/myanmar/admin_boundaries.json';
+    case 2:
+      return './data/myanmar/mmr_admin2_boundaries.json';
+    case 1:
+    default:
+      return './data/myanmar/mmr_admin1_boundaries.json';
+  }
+}
+
+export function getAdminNameProperty(adminLevel: number = 1): string {
+  switch (adminLevel) {
+    case 3:
+      return 'TS';
+    case 2:
+      return 'DT';
+    case 1:
+    default:
+      return 'ST';
+  }
+}
+
+export function fetchAdminLayerGeoJSON(
+  adminLevel = 1,
+): Promise<FeatureCollection> {
+  const url = getAdminLayerURL(adminLevel);
+  return fetch(url).then(r => r.json());
+}
```

Comment:
This doesn't scale/transfer to other countries, can you add more docstring as to what these functions are supposed to achieve?

Reply:
you are correct. I hardcoded it to start, so we can get a visible prototype we can discuss.  Before making this PR ready for review, we'll have to decide where this configuration mapping of country to admin property and file name should live.